src/displayapp/screens/WatchFaceFuturistic.h

```diff
@@ -0,0 +1,78 @@
+#pragma once
+
+#include <lvgl/src/lv_core/lv_obj.h>
+#include <chrono>
+#include <cstdint>
+#include <memory>
+#include "displayapp/screens/Screen.h"
+#include "components/datetime/DateTimeController.h"
+#include "components/battery/BatteryController.h"
+#include "components/ble/BleController.h"
+#include "components/ble/NotificationManager.h"
+#include <displayapp/screens/BatteryIcon.h>
+
+namespace Pinetime {
+  namespace Controllers {
+    class Settings;
+    class Battery;
+    class Ble;
+    class HeartRateController;
+    class NotificationManager;
+  }
+
+  namespace Applications {
+    namespace Screens {
+
+      class WatchFaceFuturistic : public Screen {
+      public:
+        WatchFaceFuturistic(DisplayApp* app,
+                            Controllers::DateTime& dateTimeController,
+                            Controllers::Battery& batteryController,
+                            Controllers::Ble& bleController,
+                            Controllers::NotificationManager& notificationManager,
+                            Controllers::HeartRateController& heartRateController);
+
+        ~WatchFaceFuturistic() override;
+
+        void Refresh() override;
+
+      private:
+        uint8_t sHour, sMinute, sSecond;
+
+        DirtyValue<std::chrono::time_point<std::chrono::system_clock, std::chrono::nanoseconds>> currentDateTime;
+        DirtyValue<uint8_t> heartbeat {};
+        DirtyValue<bool> heartbeatRunning {};
+        DirtyValue<bool> isCharging {};
+        DirtyValue<uint8_t> batteryPercentRemaining {0};
+
+        lv_style_t arc_fg_style;
+        lv_style_t arc_bg_style;
+
+        lv_obj_t* arc_sec;
+        lv_obj_t* arc_min;
+        lv_obj_t* arc_hour;
+        lv_obj_t* label;
+        lv_obj_t* heartbeatIcon;
+        lv_obj_t* heartbeatValue;
+        lv_obj_t* plugIcon;
+        lv_obj_t* batteryStatus;
+        lv_obj_t* batteryAlert;
+
+        const Controllers::DateTime& dateTimeController;
+        Controllers::Battery& batteryController;
+        Controllers::Ble& bleController;
+        Controllers::NotificationManager& notificationManager;
+        Controllers::HeartRateController& heartRateController;
+
+        // Based on the tango terminal colour theme.
+        static constexpr int innerTextColor = 0xeeeeec;
+        static constexpr int hoursArcColor = 0xef2929;
+        static constexpr int minutesArcColor = 0x729fcf;
+        static constexpr int secondsArcColor = 0x8ae234;
+        static constexpr int outerTextColor = 0xfce94f;
```

Comment:
Initializing the colors to lv_color_t with LV_COLOR_MAKE is more efficient. Move these to an anonymous namespace in the cpp file.